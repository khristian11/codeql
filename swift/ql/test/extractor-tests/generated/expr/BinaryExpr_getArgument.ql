import codeql.swift.elements
import TestUtils

from BinaryExpr x, int index
where toBeTested(x)
select x, index, x.getArgument(index)
