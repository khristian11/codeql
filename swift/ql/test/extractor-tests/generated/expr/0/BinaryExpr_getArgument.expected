| expressions.swift:17:9:17:14 | ... call to != ... | 0 | : x |
| expressions.swift:17:9:17:14 | ... call to != ... | 1 | : 0 |
| expressions.swift:44:12:44:16 | ... call to + ... | 0 | : x |
| expressions.swift:44:12:44:16 | ... call to + ... | 1 | : y |
| expressions.swift:47:12:47:16 | ... call to + ... | 0 | : x |
| expressions.swift:47:12:47:16 | ... call to + ... | 1 | : y |
| expressions.swift:49:19:49:24 | ... call to + ... | 0 | : $0 |
| expressions.swift:49:19:49:24 | ... call to + ... | 1 | : $1 |
| expressions.swift:50:12:50:17 | ... call to + ... | 0 | : $0 |
| expressions.swift:50:12:50:17 | ... call to + ... | 1 | : $1 |
| expressions.swift:66:8:66:12 | ... call to < ... | 0 | : x |
| expressions.swift:66:8:66:12 | ... call to < ... | 1 | : 0 |
